src/OSPSuite.Core/Domain/QuantityPKParameter.cs

```diff
@@ -11,10 +12,10 @@ public class QuantityPKParameter : IWithDimension, IWithName
       /// <summary>
       ///    The values. One item for each individual
       /// </summary>
-      public virtual List<float> Values { get; private set; }
+      public virtual float[] Values { get; private set; }
```

Comment:
Same here. Need to return an array (it should have been an array at the first place).
This will require a project conversion. In order to not forget, I created an issue in PKSim repo